src/examples/hotgym/HelloSPTP.cpp

```diff
@@ -171,11 +170,11 @@ Real64 BenchmarkHotgym::run(UInt EPOCHS, bool useSPlocal, bool useSPglobal, bool
 
     //Anomaly (pure x likelihood)
     tAn.start();
-    res = an.compute(outSP /*active*/, prevPred_ /*prev predicted*/);
+    res = an.compute(outSP.getDense() /*active*/, prevPred_ /*prev predicted*/);
```

Comment:
I need SDR_dense_t to be vector, not current vector.
Should I set it just there.
Or make it templated?
It seems to be a big change which breaks a lot of stuff in SDR.